app/controllers/stories_controller.rb

```diff
@@ -35,12 +35,14 @@ def show
   private
 
   def redirect_to_changed_username_profile
-    if @user = User.find_by_old_username(params[:username].tr("@", "").downcase)
+    potential_username = params[:username].tr("@", "").downcase
+    @user = User.find_by_old_username(potential_username) || User.find_by_old_old_username(potential_username)
+    @organization = Organization.find_by_old_slug(potential_username) || Organization.find_by_old_old_slug(potential_username)
```

Comment:
@benhalpern I added proper redirect for org paths. I also changed the method a bit, and not sure if this is the best approach since it technically runs 4 queries each time.